docs/0400|API.md

```diff
@@ -202,3 +202,13 @@ JSON.parse(company.get('matrix'))
 ```
 
 For more information check out the [JSON Derived Relationships blog post](https://homebase.io/blog/homebase-react-0.5.0-json-derived-relationships)
+
+##  Debugging tips
```

Comment:
I would consider nesting this under the HomebaseProvider heading

Reply:
And maybe also creating a separate troubleshooting or FAQ page if you think this should be an H2 on its own

Reply:
I'll nest it under HomebaseProvider for now but can break it out into a separate page when we add the incoming tooling.